### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by Alan-Jowett**
## Description

Fix typographical errors in the code.

## Testing

CI/CD. Shouldn't be any actual code change.

## Documentation

No.

### CODE REVIEW

File: tools/bpf2c/bpf2c_dll.c [hunk 1]
```diff
@@ -33,8 +33,8 @@ division_by_zero(uint32_t address)
     fprintf(stderr, "Divide by zero at address %d\n", address);
 }
 
-#define FIND_METADATA_ENTRTY(NAME, X) \
-    if (std::string(NAME) == #X)      \
+#define FIND_METADATA_ENTRY(NAME, X) \
```
Comment (saxena-anurag) [inline tools/bpf2c/bpf2c_dll.c:36]: Is this macro used anywhere? I could not find it being used. If it is not used, I think we can remove this altogether.

File: docs/GettingStarted.md [hunk 2]
```diff
@@ -95,9 +95,9 @@ This will build the following binaries:
                 and the EbpfCore and NetEbpfExt drivers to be loaded on a remote system to test.
 * `socket_tests.exe`: Application for testing the eBPF extension that implements the BPF_CGROUP_SOCK_ADDR program type and related attach types.
 
-and a few binaries just used for demo'ing eBPF functionality, as in the demo walkthrough discussed below:
+and a few binaries just used for demo'ing eBPF functionality, as in the demo walk-through discussed below:
```
Comment (dthaler) [inline docs/GettingStarted.md:98]: Some dictionaries do have walkthrough as unhyphenated:
* https://www.urbandictionary.com/define.php?term=walkthrough
* https://dictionary.cambridge.org/dictionary/english/walkthrough

See https://grammarhow.com/walkthrough-walk-through-or-walk-through/ for discussion that discusses both and says it's a stylistic choice.  I'd prefer reverting.   Any spell-check utility ought to have the ability to add words to the dictionary and this one should be added.

File: docs/GettingStarted.md [hunk 3]
```diff
@@ -274,7 +274,7 @@ This uses `bpf_xdp_adjust_head` helper function to encapsulate an outer IP heade
    1. Allow inbound traffic for `xdp_tests.exe` through Windows Defender Firewall. See **Note 1** below.
    2. Run `xdp_tests.exe xdp_encap_reflect_test --remote-ip <IP on the first host>`. See **Note 2** below.
 
-#### Decapsulation Test
+#### De-encapsulation Test
```
Comment (dthaler) [inline docs/GettingStarted.md:277]: Please revert.  "Decapsulation" is the industry accepted term here, not de-encapsulation.

File: docs/GettingStarted.md (hunk 2 above)
Comment (dthaler) [reply docs/GettingStarted.md]: tutorial.md still has changes to change "walkthrough" to "walk-through"

**Review by saxena-anurag — APPROVED** [review]

**Pull request merged**